### PR TITLE
Adding Canadian English as a valid translation

### LINKED ISSUE #4765: Add support for en-CA locale
Per a request from a user we'd like to add Canadian English strings -

https://make.wordpress.org/polyglots/2016/01/31/can-en-ca-be-added-to-the-android/

### PR DESCRIPTION
**Pull request by astralbodies**
Closes #4765 

Adds en-ca to the Xcode project setup, base set of translations, and updated the Ruby script to pull this locale.

Needs Review: @sendhil

### CODE REVIEW

File: Scripts/update-translations.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@
 # * German
 # * Greek
 # * English
+# * English (Canada eh)
```
Comment (frosty) [inline Scripts/update-translations.rb]: I'm not sure it's actually used for anything, but do you want to add `en-ca` to line 5 too?

**Comment by frosty** [comment]
Other than my nitpicky comment above, looks good! :shipit:

File: Scripts/update-translations.rb (hunk 1 above)
Comment (astralbodies) [inline Scripts/update-translations.rb]: Good call!

**Comment by astralbodies** [comment]
Thanks for the review @frosty!

**Pull request merged**